Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python as build
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR /app/
+
+RUN python -m venv /opt/venv
+# Enable venv
+ENV PATH="/opt/venv/bin:$PATH"
+
+COPY requirements.txt /app/
+COPY application.py /app/
+COPY templates /app/
+
+RUN pip install -Ur requirements.txt
+
+EXPOSE 5000
+ENV FLASK_APP=application.py
+#USER nobody
+CMD ["flask", "run", "--host", "0.0.0.0"]
```

Comment:
you left the new-line off on purpose because you know it burns my soul to see that little red icon?! ;)

Reply:
Honestly, I used VScode and let it do what it wants.

Reply:
All the cool kids are installing https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig

so that:
https://github.com/thekbb/shit-bruce-says/blob/main/.editorconfig#L4

Reply:
Huh. I installed editorconfig and it didn't add a newline on the Dockerfile

Reply:
Never min. Only applied on save.